backend/.env.example

```
@@ -4,3 +4,8 @@ JWT_SECRET=your_jwt_secret_here
 MONGO_URI=your_mongodb_connection_string_here
 
 PORT=4000
+
+REDIS_HOST=redis
+REDIS_PORT=6379
+REDIS_PWD=your_redis_password_if_any
+REDIS_USER=default
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)